stable_baselines3/common/vec_env/__init__.py

```diff
@@ -65,7 +67,9 @@ def sync_envs_normalization(env: "GymEnv", eval_env: "GymEnv") -> None:
     """
     env_tmp, eval_env_tmp = env, eval_env
     while isinstance(env_tmp, VecEnvWrapper):
+        assert isinstance(eval_env_tmp, VecEnvWrapper)  # Ensure that env and eval_env are similarly wrapped.
```

Comment:
I think this is a breaking change (see how it is handled in `EvalCallback`).
There are also several issues related to that.
It might be ok to add those assert but we must improve the error message then.